pkg/osquery/runtime/runtime.go

```diff
@@ -477,17 +502,41 @@ func (r *Runner) launchOsqueryInstance() error {
 		o.opts.distributedPluginFlag = "internal_noop"
 	}
 
+	// If we're on windows, ensure that we're looking for the .exe
+	if runtime.GOOS == "windows" && !strings.HasSuffix(o.opts.binaryPath, ".exe") {
+		o.opts.binaryPath = o.opts.binaryPath + ".exe"
+	}
+
+	// before we start osqueryd, check with the update system to
+	// see if we have the newest version. Do this everytime. If
+	// this prove undesirable, we can expost a function to set
```

Comment:
```suggestion
	// this prove undesirable, we can expose a function to set
```

Hey, at least you know I'm reading all this 😛